website/sidebars.json

```diff
@@ -9,7 +9,8 @@
       "research",
       "grants",
       "claims",
-      "faq"
+      "faq",
```

Comment:
this is nit picky of me, but what do you think of possible ordering this alphabetically?